spec/system/consumer/registration_spec.rb

```diff
@@ -70,6 +70,10 @@
       expect(page).to have_content 'Last step to add My Awesome Enterprise!'
 
       # Choosing a type
+      click_button "Create Profile"
+      expect(page).to have_content("Please choose one. Are you are producer?")
+      expect(page).to have_button "Create Profile", disabled: false
```

Comment:
If no profile has been chosen, then the button should be disabled I think - so disabled should be `true`here?
`expect(page).to have_button "Create Profile", disabled: true`

Reply:
Ideally, yes.
But then we would have to listen for every update made in the form to check if its valid to re-enable the button which could be expensive.  (Or could be much simpler that I am unaware of. New to ruby and front-end)
So made the changes so we just show the show the error message without disabling or proceeding, If that makes sense. 

Please let me know if I'm missing something as a ruby noob.